### DISCUSSION TITLE
Welcome to yadacoin Discussions!

### DISCUSSION THREAD

**Discussion by pdxwebdev** [pdxwebdev/yadacoin#101] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TomPelsz** [comment]
Hi all!
I’m Tom. CPU mining brought me to Yada! I was searching for low cap gem I can mine and here I am. 2y later I still think it’s super undervalued project for what it offers. Whenever I’m back to cpu mining, Yada is always on my flightsheet. Hoping more people will find out that this is a groundbreaking tech which will change crypto game!

**Reply by pdxwebdev** [reply]
Thanks Tom!
